h/accounts/schemas.py

```diff
@@ -189,6 +189,11 @@ class EditProfileSchema(CSRFSchema):
         default='',
         missing=colander.null
     )
+    emailAgain = colander.SchemaNode(
```

Comment:
If deform were rendering it....

But isn't the form hand written?
On May 15, 2015 7:01 AM, "Sean Hammond" <notifications@github.com> wrote:

> In h/accounts/schemas.py
> <https://github.com/hypothesis/h/pull/2131#discussion_r30410245>:
>
> > @@ -189,6 +189,11 @@ class EditProfileSchema(CSRFSchema):
> >          default='',
> >          missing=colander.null
> >      )
> > +    emailAgain = colander.SchemaNode(
>
>  We might not need to do any of the validation ourselves if we just set
> the widget kwarg on the email node to one of these.
>
> This results in a great big stack trace from deform.
>
> I'll try to figure it out but to be honest I feel this is a ludicrously
> complicated way to compare two strings (involving two third-party
> libraries). How many forms do we actually have in h? Wondering whether
> having deform and colander in our code at all is justifiable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2131/files#r30410245>.
>